### PR TITLE
Vm readme updates

### PR DESCRIPTION
**Pull request by jspaaks**

### CODE REVIEW

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md [hunk 1]
```diff
@@ -5,11 +5,12 @@ Describe general layout of the approach
 ## Prerequisites
 
 1. Install VirtualBox on the client: https://www.virtualbox.org/wiki/Linux_Downloads
-1. Download Ubuntu 18.04.4-desktop image from http://releases.ubuntu.com/18.04/
+1. Download an Ubuntu 18.04.4 image from http://releases.ubuntu.com/18.04/. Both the desktop and the server variant are
```
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:8]:
```suggestion
1. Download an Ubuntu 18.04 image from http://releases.ubuntu.com/18.04/. Both the desktop and the server variant are
```

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md [hunk 2]
```diff
@@ -5,11 +5,12 @@ Describe general layout of the approach
...
+suitable --choose whichever you're comfortable with.
 
 ## Server side configuration
 
-1. Create a new virtual machine in VirtualBox using Ubuntu 18.04.4-desktop as a base image.
+1. Create a new virtual machine in VirtualBox using Ubuntu 18.04.4 as a base image.
```
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:13]:
```suggestion
1. Create a new virtual machine in VirtualBox using Ubuntu as a base image.
```

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md [hunk 3]
```diff
@@ -5,11 +5,12 @@ Describe general layout of the approach
...
 1. Configure the VM with at least 2 CPUs.
 1. Configure main memory to use 4 GB.
 1. Configure video memory to use the maximum of 128 MB.
```
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:16]:
```suggestion
```

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md [hunk 4]
```diff
@@ -5,11 +5,12 @@ Describe general layout of the approach
...
 1. Install VirtualBox on the client: https://www.virtualbox.org/wiki/Linux_Downloads
-1. Download Ubuntu 18.04.4-desktop image from http://releases.ubuntu.com/18.04/
+1. Download an Ubuntu 18.04.4 image from http://releases.ubuntu.com/18.04/. Both the desktop and the server variant are
+suitable --choose whichever you're comfortable with.
 
 ## Server side configuration
 
-1. Create a new virtual machine in VirtualBox using Ubuntu 18.04.4-desktop as a base image.
+1. Create a new virtual machine in VirtualBox using Ubuntu 18.04.4 as a base image.
 1. Configure the VM with at least 2 CPUs.
```
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:14]:
```suggestion
You can keep the default settings for the new virtual machine or adjust it as you prefer.
```

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md [hunk 5]
```diff
@@ -5,11 +5,12 @@ Describe general layout of the approach
...
 1. Configure main memory to use 4 GB.
```
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:15]:
```suggestion
```

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md (hunk 3 above)
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:16]:
```suggestion
```

File: linux-ubuntu-client-to-local-machine-via-virtualbox/README.md [hunk 6]
```diff
@@ -25,14 +26,22 @@ Describe general layout of the approach
 1. Configure OpenSSH, check permissions on relevant files and directories
```
Comment (fdiblen) [inline linux-ubuntu-client-to-local-machine-via-virtualbox/README.md:26]:
```suggestion
1. Configure a ssh-server (OpenSSH) for remote connection, check permissions on relevant files and directories
```

**Comment by jspaaks** [comment]
Thanks for reviewing @fdiblen!

**Pull request merged**